platform/core/src/measurements/classes/MeasurementApi.js

```diff
@@ -7,9 +8,7 @@ import guid from '../../utils/guid';
 import studyMetadataManager from '../../utils/studyMetadataManager';
 import { measurementApiDefaultConfig } from './../configuration.js';
 
-const configuration = {
-  ...measurementApiDefaultConfig,
-};
+const configuration = measurementApiDefaultConfig;
```

Comment:
If we're not going to expand the config, spreading is safer. It prevents accidentally modifying the default config by reference.